README.md

```diff
@@ -24,276 +15,63 @@ project.
-
-We also welcome suggestions of features that you feel would be
-valuable or changes that you feel would make Zulip a better open
-source project, and are happy to support you in adding new features or
-other user experience improvements to Zulip.
-
-If you have a new feature you'd like to add, we recommend you start by
-opening a GitHub issue about the feature idea explaining the problem
-that you're hoping to solve and that you're excited to work on it.  A
-Zulip maintainer will usually reply within a day with feedback on the
-idea, notes on any important issues or concerns, and and often tips on
-how to implement or test it.  Please feel free to ping the thread if
-you don't hear a response from the maintainers -- we try to be very
-responsive so this usually means we missed your message.
-
-For significant changes to the visual design, user experience, data
-model, or architecture, we highly recommend posting a mockup,
-screenshot, or description of what you have in mind to the
-[#design](https://chat.zulip.org/#narrow/stream/design) stream on
-[chat.zulip.org][czo-doc] to get broad feedback before you spend too
-much time on implementation details.
-
-Finally, before implementing a larger feature, we highly recommend
-looking at the
-[new feature tutorial](http://zulip.readthedocs.io/en/latest/new-feature-tutorial.html)
-and [coding style guidelines](http://zulip.readthedocs.io/en/latest/code-style.html)
-on ReadTheDocs.
-
-Feedback on how to make this development process more efficient, fun,
-and friendly to new contributors is very welcome!  Just send an email
-to the [zulip-devel](#community) list with your thoughts.
-
-When you feel like you have completed your work on an issue, post your
-PR to the
-[#code review](https://chat.zulip.org/#narrow/stream/code.20review)
-stream on [chat.zulip.org][czo-doc].  This is our lightweight process
-that gives other developers the opportunity to give you comments and
-suggestions on your work.
-
-## License
-
-Copyright 2011-2017 Dropbox, Inc., Kandra Labs, Inc., and contributors
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
-The software includes some works released by third parties under other
-free and open source licenses. Those works are redistributed under the
-license terms under which the works were received. For more details,
-see the ``docs/THIRDPARTY`` file included with this distribution.
-
-
-[czo-doc]: https://zulip.readthedocs.io/en/latest/chat-zulip-org.html
+## Next steps
```

Comment:
This heading feels a bit off, but I can't think of something better.  Maybe add to the list of follow-ups "start a thread about what this should be".